array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,21 @@
+/* exported initial */
+
+function initial(array) {
+  var firstIndexes = [];
+  for (var i = 1; i < array.length; i++) {
+    firstIndexes.push(array[i - 1]);
```

Comment:
If you start the loop at `0` and end the loop one index before the last index, you can just use `i` in this loop